src/tools/compiletest/src/runtest.rs

```diff
@@ -3608,7 +3608,7 @@ fn nocomment_mir_line(line: &str) -> &str {
 
 fn read2_abbreviated(mut child: Child) -> io::Result<Output> {
     use crate::read2::read2;
-    use std::mem::replace;
+    use std::mem::take;
```

Comment:
@dtolnay Any thoughts why this doesn't compile?  I have the `mem_take` feature enabled in this crate.  Wack

Reply:
I think `compiletest` seems to be built with a stable compiler that doesn't have `take` yet.

Reply:
cc @pietroalbini

Reply:
This change will probably need to be reverted until 1.36.0 (which should be released next week I believe) as `take` was added the first day of 1.35.0.

Reply:
We intentionally build compiletest and several other tools with the bootstrap compiler (current beta).

Reply:
@czipperz Just skip the changes to compiletest for now and make them once we have a new beta compiler. To not forget, leave a note for yourself on the tracking issue. :)

Reply:
KK